hack/render-manifests.go

```diff
@@ -14,25 +15,40 @@ func exitWithError(err error, cause string, args ...interface{}) {
 	os.Exit(1)
 }
 
+const NumArgs = 5
+
+const (
+	HandlerNamespaceIdx = iota
+	HandlerImageIdx
+	HandlerPullPolicyIdx
+	InputDirIdx
+	OutputDirIdx
+)
+
 func main() {
 	type Inventory struct {
-		Namespace           string
+		HandlerNamespace    string
 		NMStateHandlerImage string
-		ImagePullPolicy     string
+		HandlerPullPolicy   string
+		HandlerPrefix       string
 	}
 
-	if len(os.Args) < 6 {
-		fmt.Fprintf(os.Stderr, "usage: go run render-manifests.go [namespace] [image] [pull-policy] [input-dir] [output-dir]\n")
+	handlerPrefix := flag.String("handler-prefix", "", "Name prefix for the handler's resources")
```

Comment:
Maybe it's time to put naming args for all of it, this is growing like crazy.

Reply:
I was thinking that same thing. That's why I did the iota!